pypgstac/pypgstac/migrations/pgstac.0.3.3-0.3.4.sql

```diff
@@ -104,17 +104,17 @@ BEGIN
 
             prev_area := unionedgeom_area;
 
-            RAISE NOTICE '% % % %', st_area(unionedgeom)/tilearea, counter, scancounter, ftime();
+            RAISE NOTICE '% % % %', unionedgeom_area/tilearea, counter, scancounter, ftime();
             IF fields IS NOT NULL THEN
                 out_records := out_records || filter_jsonb(iter_record.content, includes, excludes);
             ELSE
                 out_records := out_records || iter_record.content;
             END IF;
 
-            IF counter > _limit
+            IF counter >= _limit
```

Comment:
because we set counter to 1 by default, if we set limit to 1, we will get 2 items